crates/flipperzero/src/notification/mod.rs

```diff
@@ -0,0 +1,308 @@
+//! Furi notifications.
+
+use core::ffi::c_char;
+
+use flipperzero_sys as sys;
+use flipperzero_sys::furi::UnsafeRecord;
+
+///Default notification messages.
+pub mod messages;
+///Default notification notes.
+pub mod notes;
+///Default notification sequences.
+pub mod sequences;
```

Comment:
Rather than grouping all sequences together, the public API might be simpler if we group by kind of notification:
```suggestion
pub mod backlight;
pub mod led;
pub mod sounds;
pub mod vibro;
```

Reply:
This seems like a great idea to simplify the API

Reply:
Per my above reply, I think moving some of the single-message constants currently defined in `messages` into these "kind-of" modules would also simplify the API.